pkg/options/types.go

```diff
@@ -106,56 +105,21 @@ func (r *ResourceSet) Type() string {
 }
 
 // NodeType represents a nodeName to query from.
-type NodeType map[string]struct{}
+type NodeType string
 
-// Set converts a comma-separated string of nodename into a slice and appends it to the NodeList
-func (n *NodeType) Set(value string) error {
-	s := *n
-	cols := strings.Split(value, ",")
-	for _, col := range cols {
-		col = strings.TrimSpace(col)
-		if len(col) != 0 {
-			s[col] = struct{}{}
+// GetNodeFieldSelector returns a nodename field selector.
+func (n *NodeType) GetNodeFieldSelector(noNodeAssigned bool) string {
+	if noNodeAssigned {
```

Comment:
```suggestion
func (n *NodeType) GetNodeFieldSelector(fetchUnscheduledPods bool) string {
	if fetchUnscheduledPods {
```

nit: Function argument name was a bit unclear. It was describing the effect of the argument not the cause/condition it represented.